SpaceyaTech/CoLabs
# 18 create listing for farmdirect by fred ouko

In this branch I:
- Added the FarmDirect Project 
### How to Test
- Run ``npm start``
- Check the Project Page in Alphabetical order 
- Also check the details section of FarmDirect
